web/templates/machine/machines.html.eex

```diff
@@ -2,7 +2,7 @@
   <h1>Configured machines</h1>
 
   <div class="machine-list">
-    <%= for machine <- @machines do %>
+    <%= for %{active: true} = machine <- @machines do %>
```

Comment:
Nice trick! I will have to keep this in mind.

Reply:
But this list should show all machines so that they can be reactivated again. Maybe deactivated machines can be shown in a separate list below with only action "Activate"?

Reply:
Yeah, I thought about that. The issue said:
> Deactivation should make a machine invisible in the machines list

I think it makes sense to add a second list in the bottom.

Reply:
Yeah the comment is unfortunately ambiguous. It meant that the machine should be made invisible in the _profile page_ machines list. I should have worded it better. :)